debian/copyright

```diff
@@ -1,9 +1,19 @@
 Format: http://dep.debian.net/deps/dep5
 Upstream-Name: nest-simulator
-Source: https://github.com/nest/nest-simulator/archive/master.zip
+Source: https://github.com/nest/nest-simulator/
+
+Files: *
+Copyright: 2004-2019 The NEST Initiative <nest_user@nest-initiative.org>
```

Comment:
As a conscious policy decision, we always use 2004, the year of the first public release, as year of the NEST copyright, even for changes added later. So it should only be 2004 here.

Reply:
you can use the initial copyright, but debian uses whenever the source was started being written, to when the last change has been made to it, so 2004-2019 for debian/copyright is perfectly fine. and you shouldn't have that file at all anyways in a source tarball release, see top suggestion

Reply:
@alexmyczko Looking at https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#copyright-field, there seems no requirement to include a mail address on the copyright line, so it should be removed here. I also do not find elsewhere (https://www.debian.org/doc/debian-policy/ch-docs.html#s-copyrightfile, https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#copyright) any explicit policy on copyright dates for Debian. Therefore, we should use 2004, and only 2004 here in keeping with general NEST Copyright policy.